src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/email.yml

```diff
@@ -49,7 +49,7 @@ admin_emails_delete_all:
 
 admin_emails_delete:
     path: /delete/{mailId}
-    methods: [GET]
+    methods: [POST]
```

Comment:
It's a BC, should we accept it @eternoendless

Reply:
Fuck damn... Merged it without seeing your comment